core/frontend/meta/description.js

```diff
@@ -1,22 +1,27 @@
-var _ = require('lodash'),
-    settingsCache = require('../../server/services/settings/cache');
+const _ = require('lodash');
+const settingsCache = require('../../server/services/settings/cache');
 
 function getDescription(data, root, options) {
-    var description = '',
-        postSdDescription,
-        context = root ? root.context : null,
-        blogDescription = settingsCache.get('description');
+    const context = root ? root.context : null;
+    const siteDescription = settingsCache.get('meta_description') || settingsCache.get('description');
+
+    let description = '';
+    let postSdDescription;
 
     options = options ? options : {};
 
-    // We only return meta_description if provided. Only exception is the Blog
-    // description, which doesn't rely on meta_description.
+    // We only return meta_description if provided
     if (data.meta_description) {
         description = data.meta_description;
     } else if (_.includes(context, 'paged')) {
         description = '';
     } else if (_.includes(context, 'home')) {
-        description = blogDescription;
+        if (options && options.property) {
+            const blogSdDescription = options.property + '_description';
+            description = settingsCache.get(blogSdDescription) || '';
```

Comment:
@gargol Where does `blogSdDescription` come from? Couldn't find it in current master. Also maybe better to not use the word `blog` anymore for new changes? Also couldn't figure out where `options.property` comes from and what it might contain.

Reply:
Very good point! I even did some renames in this changeset from blog -> site and butchered it here :grimacing: 	

The naming comes from similar naming for `postSdDescription` variable which I interpreted as something like "{resource}**S**ocial?**d**ata?Description". 

Will do a rename `blogSdDescription` -> `siteSdDescription` + double check more related metadata vairable names for any more `blog` usages :+1: 

Thanks @AileenCGN!